testng-eclipse-plugin/src/main/org/testng/eclipse/ui/util/ConfigurationHelper.java

```diff
@@ -171,36 +171,18 @@ public static String getJvmArgs(ILaunchConfiguration configuration) throws CoreE
 
     addDebugProperties(jvmArgs, configuration);
 
-    switch (ConfigurationHelper.getProtocol(configuration)) {
-    case STRING:
-      jvmArgs.append(" -Dtestng.eclipse.stringprotocol");
-      break;
-    default:
-      break;
-    }
-
     return VariablesPlugin.getDefault().getStringVariableManager()
         .performStringSubstitution(jvmArgs.toString());
   }
 
   /**
-   * Pass the system properties we were called with to the RemoteTestNG process.
+   * Add the debug and/or verbose properties of RemoteTestNG if user specified
    */
   private static void addDebugProperties(StringBuilder vmArgs, ILaunchConfiguration config) {
-    String[] debugProperties = new String[] {
-        RemoteTestNG.PROPERTY_DEBUG,
-        RemoteTestNG.PROPERTY_VERBOSE
-    };
-    for (String p : debugProperties) {
-      if (System.getProperty(p) != null) {
-        vmArgs.append(" -D").append(p);

```

Comment:
let's just simplify this, so that behavior same as what user see on their launch configuration